### PR TITLE
awesomeshot: update to 1.0.5

### PR DESCRIPTION
**Pull request by nihsx**

### CODE REVIEW

**Review by Grimler91 — CHANGES_REQUESTED** [review]
Thanks, see comments.

Please squash the commits also (no need to update version in one commit and sha256sum in another)

File: packages/awesomeshot/build.sh [hunk 1]
```diff
@@ -2,10 +2,11 @@ TERMUX_PKG_HOMEPAGE=https://github.com/mayTermux/awesomeshot
...
-TERMUX_PKG_VERSION=1.0.2
+TERMUX_PKG_VERSION=1.0.5
 TERMUX_PKG_SRCURL=https://github.com/mayTermux/awesomeshot/archive/refs/tags/v.${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=ee2b73dc2714ebb6470e3ff051961d2a01783c0fb3993efda528abfb3aeb6c08
+TERMUX_PKG_SHA256=60a850a8d7015f45795622ff58d7a7445055f38f81f0bfea0e870974e454bf30
 TERMUX_PKG_DEPENDS="bash"
 TERMUX_PKG_PLATFORM_INDEPENDENT=true
 TERMUX_PKG_BUILD_IN_SRC=true
 TERMUX_PKG_EXTRA_MAKE_ARGS="PREFIX=$TERMUX_PREFIX"
+TERMUX_PKG_DEPENDS="termux-api imagemagick inotify-tools bc make"
```
[the remark this answers is not in the document]
Comment (nihsx) [reply packages/awesomeshot/build.sh]: Thanks, sorry I didn't see `TERMUX_PKG_DEPENDS` before

**Comment by Grimler91** [comment]
Please squash the commits and then we can merge this

**Comment by nihsx** [comment]
> Please squash the commits and then we can merge this

Sorry, I'm still learning git and github, is that correct?

**Comment by wmcbtech30** [comment]
`git rebase -i HEAD~n` where n is the number of recent commits you've just made. then squash commits from pick to squash

https://www.git-tower.com/learn/git/faq/git-squash

**Comment by nihsx** [comment]
> `git rebase -i HEAD~n` where n is the number of recent commits you've just made. then squash commits from pick to squash
> 
> https://www.git-tower.com/learn/git/faq/git-squash

Is it already correct?

**Pull request closed without merging**